src/schema/rules/files/deriv/imaging.yaml

```
@@ -35,6 +35,17 @@ func_volumetric:
     density: optional
     description: optional
 
+func_dtseries:
+  $ref: rules.files.raw.func.func
+  entities:
+    $ref: rules.files.raw.func.func.entities
+    space: optional
+    resolution: optional
+    density: optional
+    description: optional
+  extensions:
+    - .dtseries.nii
+
```

Should we add `.func.gii` while we're here?

```suggestion
func_dtseries:
  $ref: rules.files.raw.func.func
  entities:
    $ref: rules.files.raw.func.func.entities
    space: optional
    resolution: optional
    density: optional
    description: optional
  extensions:
    - .dtseries.nii

func_gifti:
  $ref: rules.files.raw.func.func
  entities:
    $ref: rules.files.raw.func.func.entities
    hemi: required
    space: optional
    density: optional
    description: optional
  extensions:
    - .func.gii

```